pulpcore/pulpcore/app/settings.py

```diff
@@ -168,9 +168,9 @@
     # https://docs.djangoproject.com/en/1.8/ref/settings/#databases
     'databases': {
         'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'NAME': 'pulp',
-            'USER': 'pulp',
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': '',
```

Comment:
Looks like you still need a db name here.

Reply:
yeah ... thename should be 'pulp'